### PR TITLE
Auto PR creating, GitHub Releases & Clean-ups

### PR DESCRIPTION
**Pull request by Lord-Grey**
- Support for auto Kodi-PR creating and GitHub release creation
- Clean-up
- POEditor languages support

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+]
+ignore_missing_imports = true
+
+[tool.ruff]
+select = ["B", "C", "D", "E", "F", "I", "N", "S", "U", "W"]
+ignore = ["E501", "D212", "D103", "D107"]
+target-version = "py310"
+src = ["resources"]
+
+[tool.ruff.per-file-ignores]
```
Comment (Lord-Grey) [inline pyproject.toml:22]: @sanzoghenzo Could you tell me, if for the Hyperion Capture add-on any files need to be listed here?

Comment (sanzoghenzo) [reply pyproject.toml:22]: Usually I  start without/with a blank section, then I run the linters (`pipenv run pre-commit` the first time, then `pipenv run lint`) and add files and rules codes if they are not really/easily fixable

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+[[tool.mypy.overrides]]
+module = [
+    "androguard.*",
+    "uplink",
+    "requests.*",
+    "authlib.*",
+    "pytest",
+    "pyaxmlparser.*"
+]
```
Comment (sanzoghenzo) [inline pyproject.toml]: Ooops, these modules are related to another project!
I think you can remove that section and see if the last step of `pipenv run lint` (see other comment) complains about missing imports. If it does, add that modules to the list.

Comment (Lord-Grey) [reply pyproject.toml]: Removed the modules and works

File: pyproject.toml (hunk 1 above)
Comment (Lord-Grey) [reply pyproject.toml:22]: I got the idea and either fixed small things or excluded for now.

**Comment by Lord-Grey** [comment]
@sanzoghenzo Could have a a look at the output below or even check the latest code at my branch, please?
I still get the following errors:

```
pipenv --python /usr/bin/python3.11 run pre-commit
pre-commit installed at .git/hooks/pre-commit
thomas@ubuntu2204:~/hyperion.control$ pipenv --python /usr/bin/python3.11 run lint
check toml...............................................................Passed
check yaml...............................................................Passed
fix end of files.........................................................Passed
trim trailing whitespace.................................................Passed
check for added large files..............................................Passed
mdformat.................................................................Passed
black....................................................................Passed
ruff.....................................................................Passed
mypy.....................................................................Failed
- hook id: mypy
- exit code: 2

script.service.hyperion-control/resources/lib/interfaces.py: error: Source file found twice under different module names: "lib.interfaces" and "resources.lib.interfaces"
script.service.hyperion-control/resources/lib/interfaces.py: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#mapping-file-paths-to-modules for more info
script.service.hyperion-control/resources/lib/interfaces.py: note: Common resolutions include: a) adding `__init__.py` somewhere, b) using `--explicit-package-bases` or adjusting MYPYPATH
Found 1 error in 1 file (errors prevented further checking)
```
Thanks in advance.

**Comment by sanzoghenzo** [comment]
Hi @Lord-Grey, sorry for the late response! Have you seen my PR to your fork? It should solve the issues and the others that come up after fixing this ;)

**Comment by Lord-Grey** [comment]
@sanzoghenzo Thank you for the fixes!
Unfortunately, I missed the PR :(
Looks like I need to review my notification settings.

**Pull request merged**